test/main-mocha.ts

```diff
@@ -0,0 +1,24 @@
+/**
```

Comment:
Also could we not duplicate this file as well. Could we just see if mocha was loaded `window.Mocha` and based on that load the correct test?